content/realtime/usage.textile

```diff
@@ -150,80 +150,7 @@ blang[objc,swift].
-
-  ```[objc]
-  @implementation YourClass {
-      dispatch_queue_t _ablyQueue;
-      ARTRealtime *ably;
-  }
-
-  - (void)initializeAbly {
-      _ablyQueue = dispatch_queue_create("com.example.ably", NULL);
-      [self doAblyOperation:^{
-          _ably = [ARTRealtime initWithOptions:[self ablyOptions]];
-      }];
-  }
-
-  - (void)subscribeToAbly {
-      [self doAblyOperation:^{
-          [[_ably.channels get:@"foo"] subscribe:^(ARTMessage *message) {
-              // Answer back.
-              [self doAblyOperation:^{
-                  [[_ably.channels get:@"foo"] publish:@"reply" data:@"Hi back!"];
-              }];
-          }];
-      }];
-  }
-
-  - (void)doAblyOperation:(dispatch_block_t)block {
-      // Make sure we're not already in the Ably queue! This can happen if Ably
-      // calls our callback from the same task we dispatch.
-      if (dispatch_get_current_queue() == _ablyQueue) {
-          block();
-      } else {
-          dispatch_sync(_ablyQueue, block);
-      }
-  }
-  ```
-
-  ```[swift]
-  class YourClass {
-      var ablyQueue: DispatchQueue!
-      var ably: ARTRealtime!
-
-      func initializeAbly() {
-          self.ablyQueue = DispatchQueue(label: "com.example.ably")
-          self.doAblyOperation {
-              self.ably = ARTRealtime(options: self.ablyOptions)
-          }
-      }
-
-      func subscribeToAbly() {
-          self.doAblyOperation {
-              self.ably.channels.get("foo").subscribe { message in
-                  // Answer back.
-                  self.doAblyOperation {
-                      self.ably.channels.get("foo").publish("reply", data:"Hi back!")
-                  }
-              }
-          }
-      }
-
-      func doAblyOperation(_ operation: () -> Void) {
-          // Make sure we're not already in the Ably queue! This can happen if Ably
-          // calls our callback from the same task we dispatch.
-          if (String(validatingUTF8: __dispatch_queue_get_label(nil)) == "com.example.ably") {
-              operation()
-          } else {
-              self.ablyQueue.sync(execute: operation)
-          }
-      }
-  }
-  ```
-
-  We're working on improving this situation to be more developer-friendly and less error-prone.
-
+  **Note: The iOS client libraries are thread-safe.** Check "the README":https://github.com/ably/ably-ios#thread-safety for details.
```

Comment:
This is good. Given we have older 1.* versions that are not thread-safe, do you think it would make sense to say it was thread-safe since version `1.0.8`?